src/Donations/Micros/Authentication.tsx

```diff
@@ -185,7 +209,7 @@ function VerifyEmailModal({
       BackdropProps={{
         timeout: 500,
       }}
-      disableBackdropClick
+      // disableBackdropClick
```

Comment:
enable this again

Reply:
If I enable this, there won't be any way we can close the modal.